addons/hr_holidays/tests/common.py

```diff
@@ -44,6 +44,17 @@ def setUpClass(cls):
         })
         cls.employee_emp_id = cls.employee_emp.id
 
+        if 'hr.contract' in cls.env:
+
+            employee_contract = cls.env['hr.contract'].create({
+                'name': "employee's contract",
+                'employee_id': cls.employee_emp.id,
+                'date_start': datetime.date(2010, 1, 1),
+                'date_end': False,
+                'wage': 2500
+            })
+            employee_contract.write({'state': 'open'})
+
```

Comment:
you should remove this; you have a specific module for holidays contract so just extends this class in the right module :)

Reply:
oh sorry i was not aware of that; so you're right :)